pinot-tools/src/main/java/com/linkedin/pinot/tools/admin/command/CreateSegmentCommand.java

```diff
@@ -327,7 +317,9 @@ public boolean accept(File dir, String name) {
     segmentGeneratorConfig.setOverwrite(_overwrite);
     segmentGeneratorConfig.setTableName(_tableName);
     segmentGeneratorConfig.setSegmentName(_segmentName);
-    segmentGeneratorConfig.setTimeColumnName(_timeColumnName);
+    if (_timeColumnName != null) {
```

Comment:
we don't need this, do we? we can call the setter whether or not _timeColumnName is null?